src/Altinn.Notifications.Core/Services/ContactPointService.cs

```diff
@@ -74,51 +90,73 @@ private async Task<List<Recipient>> AugmentRecipients(List<Recipient> recipients
+
+                    if (organizationContactPoints != null)
+                    {
+                        augmentedRecipients.Add(createOrgContactPoint(recipient, organizationContactPoints));
                     }
                 }
             }
 
             return augmentedRecipients;
         }
 
-        private async Task<List<UserContactPoints>> LookupContactPoints(List<Recipient> recipients)
+        private async Task<List<UserContactPoints>> LookupPersonContactPoints(List<Recipient> recipients)
         {
-            List<UserContactPoints> userContactPoints = new();
             List<string> nins = recipients
                     .Where(r => !string.IsNullOrEmpty(r.NationalIdentityNumber))
                     .Select(r => r.NationalIdentityNumber!)
                     .ToList();
 
-            Task<List<UserContactPoints>> ninLookupTask = nins.Count > 0
-             ? _profileClient.GetUserContactPoints(nins)
-             : Task.FromResult(new List<UserContactPoints>());
+            if (nins.Count == 0)
+            {
+                return new List<UserContactPoints>();
+            }
 
-            await Task.WhenAll(ninLookupTask);
+            List<UserContactPoints> contactPoints = await _profileClient.GetUserContactPoints(nins);
 
-            userContactPoints.AddRange(ninLookupTask.Result);
+            contactPoints.ForEach(contactPoint =>
+            {
+                contactPoint.MobileNumber = MobileNumberHelper.EnsureCountryCodeIfValidNumber(contactPoint.MobileNumber);
+            });
 
-            return userContactPoints;
+            return contactPoints;
         }
 
-        private async Task<List<UserContactPointAvailability>> LookupContactPointAvailability(List<Recipient> recipients)
+        private async Task<List<OrganizationContactPoints>> LookupOrganizationContactPoints(List<Recipient> recipients)
         {
-            List<UserContactPointAvailability> contactPointAvailabilityList = new();
-
-            List<string> nins = recipients
-                    .Where(r => !string.IsNullOrEmpty(r.NationalIdentityNumber))
-                    .Select(r => r.NationalIdentityNumber!)
-                    .ToList();
-
-            Task<List<UserContactPointAvailability>> ninLookupTask = nins.Count > 0
-             ? _profileClient.GetUserContactPointAvailabilities(nins)
-             : Task.FromResult(new List<UserContactPointAvailability>());
+            /* the output from this function should include an AUHTORIZED list of user registered contact points if notification has a service affiliation 
+                will require the extension of the OrganizationContactPoints class */
+            List<string> orgNos = recipients
+             .Where(r => !string.IsNullOrEmpty(r.OrganizationNumber))
+             .Select(r => r.OrganizationNumber!)
+             .ToList();
+
+            if (orgNos.Count == 0)
+            {
+                return [];
```

Comment:
Should we use explicit type here? It's explicit a bit further up.

Reply:
rather changed the other function to return [] as well. Shorter notation and I think return type is implicit.